src/Lykke.Logs/LykkeLogToAzureSlackNotificationsManager.cs

```diff
@@ -137,5 +151,26 @@ private HashSet<string> DefaultLogLevelsInit()
                 LykkeLogToAzureStorage.MonitorType,
             };
         }
+
+        private bool IsSameMessage(LogLevel level, string message)
+        {
+            var now = DateTime.UtcNow;
+            if (_lastTimes.TryGetValue(level, out DateTime lastTime))
+            {
+                if (_lastMessages.TryGetValue(level, out string lastMessage))
```

Comment:
GetOrAdd is safe here
https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentdictionary-2.getoradd?view=netframework-4.7.2